## 1.0 Strategic Recommendations Framework

- Objective:
  - Translate analytical findings into actionable business strategies
  - Focus on revenue growth, operational efficiency, and demand optimization

- Approach:
  - Align recommendations with:
    - Demand patterns (peak vs off-peak)
    - Operational constraints (kitchen + cashier throughput)
    - Revenue drivers (AOV and order volume)

- Key Principle:
  - Maximize revenue without adding unnecessary operational complexity

## 1.1 Core Revenue Drivers

- Primary Drivers Identified:

  - Order Volume:
    - Main contributor to total revenue
    - Strongly influenced by time-of-day and day-of-week patterns

  - Average Order Value (AOV):
    - ~30.70 baseline
    - Key lever for margin-efficient revenue growth

- Key Insight:
  - Revenue growth is driven more by increasing order count and ticket size than price changes

- Implication:
  - Strategy should focus on:
    - Increasing AOV
    - Improving throughput during peak hours
    - Expanding demand during off-peak hours

## 1.2 Demand Patterns

- Peak Demand:
  - 4pm–7pm
  - Highest order concentration
  - Capacity-constrained period

- Weekend Effect:
  - Higher order volume compared to weekdays
  - Strong dinner demand

- Off-Peak Period:
  - Before 4pm
  - Lower order volume
  - Underutilized capacity

- Key Insight:
  - Demand is predictable and time-dependent

- Implication:
  - Different strategies are required for different time windows

## 1.3 Order Behavior Insights

- Order Size Patterns:
  - Solo / small orders dominate off-peak
  - Larger, multi-item orders dominate peak hours

- Menu Behavior:
  - Pizza size upgrades (Medium → Large) represent natural AOV growth opportunities
  - Add-ons (garlic bread, wings, drinks) are key upsell drivers

- Key Insight:
  - Customers follow predictable ordering patterns based on time and group size

- Implication:
  - Strategies should align with:
    - Solo ordering behavior (off-peak)
    - Family / group ordering behavior (peak)

## 1.4 Pricing & Discount Observations

- Findings:
  - Minimal discount usage observed in current operations
  - Pricing remains relatively stable across orders

- Key Insight:
  - Revenue is not currently driven by promotions or discounting

- Risk:
  - Unstructured discounts could erode margins without increasing volume

- Opportunity:
  - Introduce controlled, targeted promotions:
    - Time-based discounts (off-peak only)
    - Spend-threshold incentives
    - Bundled pricing

- Implication:
  - Discounts should be used strategically, not broadly

## 1.5 Strategic Opportunity Areas

- AOV Expansion:
  - Increase revenue per order through:
    - Bundles
    - Upsells
    - Spend thresholds

- Peak Throughput Optimization:
  - Increase orders fulfilled during 4pm–7pm
  - Focus on:
    - Kitchen efficiency
    - Cashier capacity

- Off-Peak Demand Generation:
  - Increase order volume before 4pm
  - Use:
    - Promotions
    - Combos
    - Marketing (flyers)

- Key Insight:
  - Revenue growth requires a combination of:
    - Higher AOV
    - Higher throughput
    - Better demand distribution

- Strategic Direction:
  - Align pricing, promotions, and operations with demand patterns

## 1.6 Operational Constraints & Capacity Insights

- Objective:
  - Identify operational bottlenecks limiting revenue growth
  - Connect demand patterns to real-world staffing and kitchen constraints

## 1.6.1 Peak Hour Constraint (4pm–7pm)

- Key Insight:
  - Highest order volume occurs during 4pm–7pm window
  - Demand exceeds kitchen throughput capacity

- Observations:
  - Orders cluster tightly within this time range
  - Kitchen becomes the limiting factor, not demand

- Implication:
  - Revenue is being capped by how many orders can be fulfilled, not how many customers want to order

## 1.6.2 Front-of-House Bottleneck (Cashiers)

- Key Insight:
  - Cashier throughput becomes a constraint during peak periods

- Observations:
  - Order intake slows during rush periods
  - Long wait times reduce order conversion and customer experience

- Real-World Context:
  - 1 cashier is insufficient during peak
  - 2 cashiers is minimum, but still strained during high-volume periods

- Implication:
  - Lost revenue may occur before orders even reach the kitchen

## 1.6.3 Kitchen Throughput Limitation

- Key Insight:
  - Kitchen capacity is the primary constraint on revenue during peak hours

- Observations:
  - Cooking, prep, and assembly times limit total orders per hour
  - Increased demand cannot be fulfilled without operational adjustments

- Implication:
  - Adding demand without increasing throughput will not increase revenue
  - Efficiency improvements directly translate into revenue gains

## 1.6.4 Off-Peak Underutilization

- Key Insight:
  - Significant idle capacity exists before 4pm

- Observations:
  - Low order volume during lunch and early afternoon
  - Staff and kitchen resources are underutilized

- Implication:
  - Opportunity to increase revenue without increasing costs
  - Off-peak demand is a key growth lever

## 1.6.5 Strategic Implications

- Peak Hours:
  - Focus on throughput and efficiency
  - Increase staffing (kitchen + cashiers)
  - Avoid promotions that slow operations

- Off-Peak Hours:
  - Focus on demand generation
  - Use promotions and marketing to increase order volume

- Overall:
  - Align operations with demand patterns
  - Revenue growth depends on both:
    - Increasing AOV
    - Increasing throughput

## 1.7 AOV Optimization Strategy — Tactical Implementation

- Objective:
  - Translate AOV insights into actionable tactics
  - Increase revenue per order without adding operational strain

- Core Principle:
  - Drive higher spend per transaction while maintaining speed and margin

## 1.7.1 Strategic Framework

- Key Insight:
  - Current AOV ≈ $30.70
  - Small increases in AOV produce significant revenue impact

- Approach:
  - Use structured incentives instead of blanket discounts
  - Align pricing tactics with operational constraints

## 1.7.2 Spend Threshold Incentives (Menu-Aligned)

- Strategy:
  - Use realistic price jumps based on pizza sizing and add-ons

- Example (Based on Menu Behavior):

  - Spend 25 → free garlic bread  
    (Typical: Medium Pizza ≈ 22–25 → encourages add-on)

  - Spend 35 → free 2-liter soda  
    (Typical: Large Pizza + side → pushes into full meal)

  - Spend 45 → free wings or bundle upgrade  
    (Typical: Family-size orders)

- Why It Works:
  - Aligns with natural ordering tiers:
    - Solo → Medium
    - Couple → Large + side
    - Family → multi-item order

- Impact:
  - Converts borderline orders into higher AOV tiers

## 1.7.3 Bundle Optimization (Menu-Specific)

- Strategy:
  - Build bundles around your highest-frequency items

- Example Bundles:

  - Family Deal:
    - 1 Large Specialty Pizza + Garlic Bread + 2-Liter → 31–35 range

  - Double Pizza Deal:
    - 2 Medium Pizzas → 32–36 range

  - Party Bundle:
    - 2 Large Pizzas + Wings + 2-Liter → 50–55 range

- Why It Works:
  - Mirrors real ordering behavior (groups, families)
  - Simplifies decision-making
  - Increases total ticket size

- Operational Benefit:
  - Predictable orders = faster kitchen execution

## 1.7.4 Default Upsells (Menu-Driven)

- Strategy:
  - Focus upsells on high-margin, easy-add items

- Best Upsell Targets:

  - Garlic Bread:
    - “Add garlic bread for 5?”

  - Size Upgrade:
    - “Upgrade to large for 3 more?”

  - Wings:
    - “Add wings for 8–10?”

  - Drinks:
    - “Add a 2-liter for 3–4?”

- Why It Works:
  - Low friction additions
  - Minimal impact on kitchen complexity
  - High contribution margin items

- Expected Impact:
  - 5–10% AOV increase through consistent execution

## 1.7.5 Menu Anchoring (Pricing Psychology)

- Strategy:
  - Structure pricing tiers to encourage natural upgrades

- Example (Menu Behavior):

  - Medium Pizza → ~22–25  
  - Large Pizza → ~26–29  

- Insight:
  - Small price gaps (3–4 difference) drive customers to choose larger sizes

- Why It Works:
  - Customers perceive higher value in upgrading
  - Increases AOV without additional items or complexity

- Impact:
  - Shifts order mix toward higher-value items

## 1.7.6 Strategic Takeaway

- AOV growth should be driven by:
  - Spend thresholds
  - Bundles
  - Upsells
  - Pricing structure

- Avoid:
  - Blanket discounts
  - Free items without conditions

- Key Principle:
  - Increase revenue per order without increasing operational strain

- Result:
  - Higher AOV
  - Improved margins
  - Faster, more efficient ordering flow

## 1.8 Time-Based Promotion Strategy — Peak vs Off-Peak Optimization

- Objective:
  - Align promotions with demand patterns to maximize revenue and operational efficiency
  - Apply different tactics based on time-of-day demand and capacity constraints

## 1.8.1 Demand Windows

- Off-Peak:
  - Before 4pm (low demand, excess capacity)

- Peak:
  - 4pm–7pm (high demand, capacity constrained)

- Post-Peak:
  - After 7pm (moderate demand, declining traffic)

## 1.8.2 Off-Peak Strategy (Demand Generation)

- Objective:
  - Increase order volume during low-demand hours
  - Utilize idle kitchen and labor capacity

- Example Promotions:

  - Lunch Personal Combo:
    - Personal Pizza (Cheese or Pepperoni) + Drink → 11.99

  - Slice Combo:
    - 2 Slices + Drink → 9.99–10.99

  - Small Order Incentive:
    - “3 off 20+ (11am–3pm)”

  - Flyer Offer:
    - “5 off 25 (Mon–Thurs only)”

- Why It Works:
  - Matches solo / small-group demand behavior
  - Keeps kitchen simple during slower hours
  - Converts low-value orders into mid-tier orders

## 1.8.3 Peak Strategy (Revenue Maximization)

- Objective:
  - Maximize AOV during 4pm–7pm dinner rush
  - Target families and group orders

- Example Promotions:

  - Family Bundle:
    - 1 Large Pizza + Garlic Bread + 2-Liter Soda → 31.99–34.99

  - Double Pizza Deal:
    - 2 Medium Pizzas + 2-Liter Soda → 34.99–36.99

  - Party Bundle:
    - 2 Large Pizzas + Wings + 2-Liter → 49.99–54.99

  - Spend Threshold Incentives:
    - Spend 30 → free garlic bread
    - Spend 40 → free 2-liter soda

  - Default Upsell Prompts:
    - “Upgrade to large for 3 more?”
    - “Add wings for 8–10?”

- Avoid:
  - Discounts during peak hours
  - Free items without spend requirement

- Why It Works:
  - Aligns with high-demand dinner behavior
  - Increases revenue per order without slowing throughput

## 1.8.4 Post-Peak Strategy

- Objective:
  - Maintain steady order flow after peak hours (after 7pm)
  - Capture late-night and smaller orders

- Example Promotions:

  - Late Night Add-On:
    - “Free soda with any large pizza after 7pm”

  - Wings Add-On Incentive:
    - “Add wings for 2 off after 7pm”

  - Light Bundle:
    - Medium Pizza + Garlic Bread → 22.99–24.99

- Why It Works:
  - Encourages continued ordering without overwhelming kitchen
  - Focuses on add-ons rather than heavy discounts

## 1.8.5 Strategic Takeaway

- Off-Peak:
  - Use discounts and combos to drive volume

- Peak:
  - Use bundles and upsells to increase AOV
  - Do not use discounts

- Post-Peak:
  - Use light incentives to extend demand

- Result:
  - Higher total daily revenue
  - Better labor utilization
  - Improved operational flow

# 2.0 Quantifying Revenue Opportunities

- Objective:
  - Translate strategic recommendations into measurable revenue impact
  - Estimate potential upside from key initiatives:
    - AOV expansion
    - Peak-hour throughput improvement
    - Off-peak demand growth

- Approach:
  - Use baseline metrics from the cleaned 2025 full-year order dataset
  - Apply simple scenario analysis
  - Prioritize directional business value over unnecessary modeling complexity

- Output of Section 2:
  - Estimated revenue lift by strategic lever
  - Reusable baseline metrics for later sections
  - Clear comparison of highest-ROI opportunities

## 2.1 AOV Expansion — Revenue Impact Estimation

- Objective:
  - Estimate revenue lift from increasing Average Order Value (AOV)
  - Quantify the impact of bundles, upsells, and spend-threshold incentives

## 2.1.0 Load Order Data

- Using pre-concatenated full-year dataset
- Required columns:
  - order_date
  - order_id
  - order_total

- Ensure:
  - Column names are standardized
  - Order Total is numeric
  - No missing core fields

In [16]:
import pandas as pd

# Load dataset
df = pd.read_csv("../data/cleaned/orders_2025_full.csv")

# Standardize column names
df.columns = df.columns.str.strip().str.lower().str.replace(" ", "_")

# Keep required columns for Section 2
df = df[["order_date", "order_id", "order_total"]].copy()

# Ensure numeric order total
df["order_total"] = pd.to_numeric(df["order_total"], errors="coerce")

# Drop rows missing core fields
df = df.dropna(subset=["order_date", "order_id", "order_total"])

df.head()

,order_date,order_id,order_total
0,2025-03-31 22:54:00-07:00,QQBMWE3NMZQ0Y,25.99
1,2025-03-31 22:35:00-07:00,TH041BMRDC2Y6,25.13
2,2025-03-31 22:33:00-07:00,06KJV691ZMV48,17.90
3,2025-03-31 22:24:00-07:00,ASKA2CPN7GT2A,40.69
4,2025-03-31 22:23:00-07:00,GK5Z5D128NA1M,20.05


## 2.1.0a Create Datetime & Time Features

- Convert raw timestamp into localized datetime
- Standardize timezone to America/Los_Angeles
- Create reusable time-based fields for downstream analysis:
  - hour
  - day of week
  - weekend flag
  - peak / off-peak flags

- This supports later sections without rebuilding the pipeline

In [17]:
# Create localized datetime
df["order_datetime"] = pd.to_datetime(df["order_date"], utc=True)
df["order_datetime"] = df["order_datetime"].dt.tz_convert("America/Los_Angeles")

# Create reusable time features
df["hour"] = df["order_datetime"].dt.hour
df["order_dow"] = df["order_datetime"].dt.day_name()
df["is_weekend"] = df["order_dow"].isin(["Saturday", "Sunday"])

# Create reusable business flags
df["peak_flag"] = (df["hour"] >= 16) & (df["hour"] < 19)
df["off_peak_flag"] = df["hour"] < 16

df[["order_datetime", "hour", "order_dow", "is_weekend", "peak_flag", "off_peak_flag"]].head()

,order_datetime,hour,order_dow,is_weekend,peak_flag,off_peak_flag
0,2025-03-31 22:54:00-07:00,22,Monday,False,False,False
1,2025-03-31 22:35:00-07:00,22,Monday,False,False,False
2,2025-03-31 22:33:00-07:00,22,Monday,False,False,False
3,2025-03-31 22:24:00-07:00,22,Monday,False,False,False
4,2025-03-31 22:23:00-07:00,22,Monday,False,False,False


## 2.1.0b Create Core Baseline Metrics

- Compute reusable baseline metrics for Section 2:
  - Average Order Value (AOV)
  - Total Orders

- These metrics will be reused across all revenue scenario calculations

In [18]:
baseline_aov = df["order_total"].mean()
total_orders = df["order_id"].nunique()

baseline_aov, total_orders

(30.69616119000931, 50487)

## 2.1.1 Baseline Metrics

- Average Order Value (AOV): $30.70
- Total Orders (Annual): 50,487

- These metrics form the foundation for AOV scenario modeling

In [19]:
baseline_aov = round(baseline_aov, 2)
total_orders = int(total_orders)

baseline_aov, total_orders

(30.7, 50487)

## 2.1.2 AOV Expansion Scenarios

- Scenario Design:
  - Conservative Case: +5% AOV
  - Moderate Case: +10% AOV
  - Aggressive Case: +15% AOV

- Key Insight:
  - Even small increases in AOV generate meaningful revenue impact
  - No additional labor or capacity is required
  - This is a high-margin growth lever

In [20]:
aov_scenarios = {
    "5% Increase": 0.05,
    "10% Increase": 0.10,
    "15% Increase": 0.15
}

aov_results = []

for label, pct in aov_scenarios.items():
    new_aov = baseline_aov * (1 + pct)
    revenue_lift = (new_aov - baseline_aov) * total_orders

    aov_results.append({
        "Scenario": label,
        "New AOV ($)": round(new_aov, 2),
        "Revenue Lift ($)": round(revenue_lift, 2)
    })

aov_results_df = pd.DataFrame(aov_results)

aov_results_df

,Scenario,New AOV ($),Revenue Lift ($)
0,5% Increase,32.24,77497.55
1,10% Increase,33.77,154995.09
2,15% Increase,35.30,232492.64


## 2.1.3 Interpretation

- Key Findings:
  - AOV growth creates substantial top-line revenue lift
  - This is the most operationally efficient revenue lever in the workbook

- Business Insight:
  - Upsells, bundles, and spend-threshold incentives can increase revenue without requiring more orders
  - Compared with staffing-based levers, this is lower complexity and higher margin

- Strategic Takeaway:
  - AOV expansion should be prioritized as a first-line revenue strategy

## 2.2 Peak Throughput — Revenue Impact Estimation

- Objective:
  - Quantify revenue gains from improving throughput during peak hours (4pm–7pm)
  - Estimate how many additional orders can be fulfilled with better staffing and kitchen efficiency

## 2.2.0 Peak Hour Demand Context

- Key Insight:
  - Peak demand occurs between 4pm–7pm
  - Kitchen capacity limits total orders fulfilled
  - Revenue is constrained by throughput, not demand

- Assumption:
  - Additional staffing and better kitchen execution increase fulfilled orders per hour

## 2.2.1 Baseline Peak Metrics

- Measure current order volume during peak hours
- Use full-year local-time order data
- Establish baseline revenue per peak hour

In [21]:
# Filter peak-hour orders
peak_df = df[df["peak_flag"]].copy()

# Baseline peak metrics
peak_orders_per_hour = peak_df.groupby("hour")["order_id"].nunique().mean()
peak_revenue_per_hour = peak_orders_per_hour * baseline_aov

peak_orders_per_hour, peak_revenue_per_hour

(6096.333333333333, 187157.43333333332)

## 2.2.2 Throughput Improvement Scenarios

- Scenario Design:
  - Conservative: +2 additional orders per hour
  - Moderate: +5 additional orders per hour
  - Aggressive: +8 additional orders per hour

- Assumption:
  - Improved staffing and kitchen flow increase fulfilled orders during the 4pm–7pm window

- Modeling Note:
  - Peak window assumed to last 3 hours per day

In [22]:
throughput_scenarios = {
    "Conservative (+2 orders/hr)": 2,
    "Moderate (+5 orders/hr)": 5,
    "Aggressive (+8 orders/hr)": 8
}

throughput_results = []
peak_hours_per_day = 3

for label, extra_orders in throughput_scenarios.items():
    incremental_revenue_per_hour = extra_orders * baseline_aov
    daily_revenue_lift = incremental_revenue_per_hour * peak_hours_per_day
    annual_revenue_lift = daily_revenue_lift * 365

    throughput_results.append({
        "Scenario": label,
        "Extra Orders/Hour": extra_orders,
        "Revenue Lift / Hour ($)": round(incremental_revenue_per_hour, 2),
        "Revenue Lift / Day ($)": round(daily_revenue_lift, 2),
        "Revenue Lift / Year ($)": round(annual_revenue_lift, 2)
    })

throughput_df = pd.DataFrame(throughput_results)

throughput_df

,Scenario,Extra Orders/Hour,Revenue Lift / Hour ($),Revenue Lift / Day ($),Revenue Lift / Year ($)
0,Conservative (+2 orders/hr),2,61.4,184.2,67233.0
1,Moderate (+5 orders/hr),5,153.5,460.5,168082.5
2,Aggressive (+8 orders/hr),8,245.6,736.8,268932.0


## 2.2.3 Interpretation

- Key Findings:
  - Small increases in peak-hour throughput create meaningful revenue upside
  - Peak hours represent the highest-return operating window for staffing improvements

- Business Insight:
  - This lever depends on operational execution, not pricing
  - Revenue upside is repeatable because peak demand is consistent and predictable

- Strategic Takeaway:
  - Improving kitchen throughput during 4pm–7pm is one of the clearest revenue unlocks in the business

## 2.3 Off-Peak Demand — Revenue Impact Estimation

- Objective:
  - Quantify revenue opportunities from increasing demand during off-peak hours
  - Measure the impact of promotions, flyers, and weekday demand-shaping strategies

## 2.3.0 Off-Peak Demand Context

- Core Insight:
  - Significant unused capacity exists outside peak hours
  - Growing demand in these hours can increase revenue without adding major infrastructure

- Off-Peak Definition:
  - Hours before 4pm

- Assumption:
  - Marketing efforts can generate incremental off-peak orders without disrupting peak operations

## 2.3.1 Baseline Off-Peak Metrics

- Measure current order volume during off-peak hours
- Use existing off-peak flag created earlier in the pipeline
- Establish baseline for demand-expansion scenarios

In [23]:
# Filter off-peak orders
off_peak_orders = df[df["off_peak_flag"]].copy()

# Baseline off-peak metrics
off_peak_orders_per_hour = off_peak_orders.groupby("hour")["order_id"].nunique().mean()
off_peak_revenue_per_hour = off_peak_orders_per_hour * baseline_aov

off_peak_orders_per_hour, off_peak_revenue_per_hour

(2276.4285714285716, 69886.35714285714)

## 2.3.2 Off-Peak Demand Scenarios

- Scenario Design:
  - Conservative: +2 additional orders per hour
  - Moderate: +4 additional orders per hour
  - Aggressive: +6 additional orders per hour

- Demand Drivers:
  - Flyers
  - Lunch specials
  - Time-based promotions
  - Weekday traffic campaigns

- Modeling Note:
  - Off-peak window assumed to cover 8 hours per day

In [24]:
off_peak_scenarios = {
    "Conservative (+2 orders/hr)": 2,
    "Moderate (+4 orders/hr)": 4,
    "Aggressive (+6 orders/hr)": 6
}

off_peak_results = []
off_peak_hours_per_day = 8

for label, extra_orders in off_peak_scenarios.items():
    incremental_revenue_per_hour = extra_orders * baseline_aov
    daily_revenue_lift = incremental_revenue_per_hour * off_peak_hours_per_day
    annual_revenue_lift = daily_revenue_lift * 365

    off_peak_results.append({
        "Scenario": label,
        "Extra Orders/Hour": extra_orders,
        "Revenue Lift / Hour ($)": round(incremental_revenue_per_hour, 2),
        "Revenue Lift / Day ($)": round(daily_revenue_lift, 2),
        "Revenue Lift / Year ($)": round(annual_revenue_lift, 2)
    })

off_peak_results_df = pd.DataFrame(off_peak_results)

off_peak_results_df

,Scenario,Extra Orders/Hour,Revenue Lift / Hour ($),Revenue Lift / Day ($),Revenue Lift / Year ($)
0,Conservative (+2 orders/hr),2,61.4,491.2,179288.0
1,Moderate (+4 orders/hr),4,122.8,982.4,358576.0
2,Aggressive (+6 orders/hr),6,184.2,1473.6,537864.0


## 2.3.3 Interpretation

- Key Findings:
  - Off-peak demand represents meaningful untapped revenue potential
  - This lever monetizes existing idle capacity during slower hours

- Business Insight:
  - Marketing and promotions can directly translate into incremental revenue during underutilized periods
  - Compared with peak-hour scaling, this lever carries lower operational risk

- Strategic Takeaway:
  - Off-peak demand shaping is one of the most scalable ways to smooth volume and improve total daily revenue

## 3.0 Executive Summary

- Objective:
  - Present key findings, revenue opportunities, and operational recommendations
  - Provide a clear, decision-ready summary based on full analysis

- Scope:
  - Based on 2025 full-year order data
  - Covers demand patterns, revenue drivers, and operational constraints

## 3.1 Key Findings

- Revenue is primarily driven by order volume, not pricing changes

- Peak demand is concentrated between 4pm–7pm
  - This window is capacity-constrained

- Weekends consistently outperform weekdays
  - Higher order volume and larger ticket sizes

- Off-peak hours (before 4pm) show significant underutilized capacity

- Average Order Value (~30.70) represents a major untapped growth lever

- Operational bottlenecks exist in:
  - Kitchen throughput
  - Cashier capacity during peak hours

## 3.2 Revenue Opportunities (Ranked)

- 1. Peak Throughput Optimization:
  - Increase orders fulfilled during 4pm–7pm
  - Highest revenue-per-hour impact

- 2. AOV Expansion:
  - Increase average ticket size through:
    - Bundles
    - Upsells
    - Spend thresholds
  - High-margin growth lever

- 3. Off-Peak Demand Generation:
  - Increase order volume during low-demand hours
  - Monetize existing idle capacity

- Key Insight:
  - Maximum revenue growth comes from combining all three levers

## 3.3 Operational Recommendations

- Staffing:

  - Peak Hours:
    - Increase kitchen staffing
    - Maintain minimum 2+ cashiers

  - Off-Peak:
    - Reduce staffing to match demand
    - Maintain efficiency

- Kitchen Operations:

  - Pre-prep high-demand ingredients before peak
  - Standardize workflows for top-selling items
  - Assign clear roles (prep, cook, assembly)

- Ordering Flow:

  - Prioritize speed during peak hours
  - Use bundles to reduce decision time

## 3.4 Pricing & Promotion Strategy

- AOV Strategy:

  - Use spend thresholds:
    - Spend 30 → free garlic bread
    - Spend 40 → free 2-liter soda

  - Implement bundles aligned with menu behavior:
    - Family bundles
    - Multi-pizza deals

  - Standardize upsells:
    - Garlic bread, wings, drinks

- Time-Based Promotions:

  - Off-Peak:
    - 5–10% discount
    - Lunch combos
    - Flyer campaigns

  - Peak:
    - No discounts
    - Focus on AOV and speed

- Key Principle:
  - Promotions should drive behavior, not reduce margin

## 3.5 Business Impact

- Revenue Growth:

  - AOV Expansion:
    - Direct increase in revenue per order

  - Peak Throughput:
    - Higher revenue per hour during highest demand window

  - Off-Peak Demand:
    - Increased total daily revenue without major cost increases

- Cost Efficiency:

  - Better labor alignment reduces cost per order
  - Improved utilization of existing resources

- Operational Stability:

  - Predictable demand allows structured planning
  - Standardized processes improve consistency

## 3.6 Final Takeaway

- Pizza House does not need:
  - Major price increases
  - Complex modeling systems

- Pizza House does need:
  - Alignment between demand and operations
  - Structured AOV strategy
  - Targeted, time-based promotions

- Core Strategy:

  - Peak Hours:
    - Maximize throughput and AOV

  - Off-Peak:
    - Drive demand through promotions

- Result:
  - Scalable revenue growth
  - Improved efficiency
  - Stronger, more predictable performance